Mathlib/CategoryTheory/Adjunction/Cat.lean

```diff
@@ -0,0 +1,52 @@
+/-
+Copyright (c) 2024 Nicolas Rolland. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Nicolas Rolland
+-/
+import Mathlib.CategoryTheory.Category.Cat
+import Mathlib.CategoryTheory.Adjunction.Basic
+/-!
+# Adjunctions related to Cat, the category of categories
+
+The embedding `Type ⥤ Cat` has a right adjoint `Cat.objects` mapping
+each category to its set of objects.
+
+
+## Notes
+All this could be made with 2-functors
+
+## TODO
+The embedding `Type ⥤ Cat` has a left adjoint `Cat.connectedComponents` mapping
+each category to its set of connected components.
+-/
```

Comment:
I made some suggestions to rephrase the module docstring in my first comment. Could you take a look at those and try to incorporate them?